contracts/taskModules/TaskModules.sol

```diff
@@ -0,0 +1,52 @@
+// // SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.12;
+
+import {DataTypes} from "../libraries/DataTypes.sol";
+import {OpsStorage} from "../OpsStorage.sol";
+import {TimeModule} from "./TimeModule.sol";
+import {SingleExecModule} from "./SingleExecModule.sol";
+
+contract TaskModules is TimeModule, SingleExecModule, OpsStorage {
+    constructor(address _taskTreasury, address _opsProxyFactory)
+        OpsStorage(_taskTreasury, _opsProxyFactory)
+    {}
+
+    function _handleTaskModuleOnCreate(
```

Comment:
Write this function in such a way that we dont have to update this part every single time we add a new module. It should just call into a whitelisted module with some data and there we will do the rest.